src/map/modifier.h

```diff
@@ -994,7 +995,7 @@ enum class Mod
     // 570 through 825 used by WS DMG mods these are not spares.
     //
     // SPARE IDs:
-    // 138 to 143
+    // 141 to 143
```

Comment:
👍

Reply:
yea _somebody_ didn't increment last time (it's me, hi, I'm the problem, it's me)